### ISSUE TITLE
[email feedback] storing multiple values in lists

### ISSUE THREAD

**Issue by ErinBecker** [swcarpentry/python-novice-inflammation#680]
In the section Turn a String Into a List, I feel the following code
```py
binomial_name = "Drosophila melanogaster"
group = binomial_name[0:10]
print("group:", group)

species = binomial_name[11:24]
print("species:", species)
```
could be replaced by
```py
binomial_name = "Drosophila melanogaster"
group = binomial_name[0:10]
print("group:", group)

species = binomial_name[11:23]
print("species:", species)
```

As string `binomial_name` has `"Drosophila melanogaster"`
indexed from 0 to 22, _i.e._ 23 characters. Thus the slice I feel 
should be from `[11:23]` and not `[11:24]`.

However if you write 
```py
species = binomial_name[11:24]
```
you can include the comment
```py
# an index that is too big is truncated down to the string length
```

**Comment by maxim-belkin** [comment]
Thank you for relaying this feedback to us, Erin. This issue has been addressed in #678 and previously reported in #667 and #676.

**Comment by ErinBecker** [comment]
Great! Thanks for confirming @maxim-belkin!
